export const errorMessages = {
  unexpectedError: 'Falha inesperada, tente novamente',
  fillInCorrectly: 'Por favor, verifique os dados inseridos e tente novamente.',
  networkError: 'Não foi possível conectar ao servidor, verifique sua conexão',
  notAuthorized: 'Você não tem permissão para acessar este recurso.',
  notAuthenticated: 'Faça login para acessar este recurso.',
  resourceNotFound: 'O recurso solicitado não foi encontrado.',
  tryAgainLater: 'Tente novamente mais tarde.',
  unavailableService:
    'Serviço temporariamente indisponível, tente novamente mais tarde.',
};
